### ISSUE TITLE
The Hipster Jesus API no longer exists

### ISSUE THREAD

**Issue by skylerwshaw** [kjbrum/alfred-hipsum#1]
This Alfred workflow has been broken for 6+ months due to http://hipsterjesus.com/api being out of commission. One replacement to the hipster could be the hippy: http://www.hippieipsum.me/api/v1/get/5

**Comment by kjbrum** [comment]
I noticed it stopped working a while back and haven't gotten around to updating this repo.

It would be nice to get the source code from @boogah, or at least the list of words/phrases. Wouldn't be too difficult to get a new API spun up again.

**Comment by boogah** [comment]
I never ran the HipsterJesus API, so I can't share anything WRT that. However, I've got some good news...

Ever since I switched [Hipsum](https://hipsum.co/) from using my poorly maintained custom code to the Any Ipsum WordPress plugin — which was years ago at this point — there's [an official API endpoint](https://hipsum.co/api/?type=hipster-centric&start-with-lorem=1) already available.

The only reason I hadn't (publicly) mentioned it until now is because I didn't want to stomp on folks who liked my project enough to build their own API services for folks. But if HJ is dead, I guess I'm okay to share it.

The parameters for the official API are as follows:

* `type` — `hipster-centric` for hipster speak only or `hipster-latin` for hipster speak mixed with miscellaneous latin.
* `paras` — Number of paragraphs. Defaults to 5. _Optional_
* `sentences` — Number of sentences. This parameter overrides `paras`. _Optional_
* `start-with-lorem` Pass `1` to start the first paragraph with "I'm baby ". _Optional_

Sorry that your extension broke! 😬 Hope all of this helps!
